packages/experimental/RadioGroup/src/Radio/RadioExperimental.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { Radio } from './Radio';
+import * as renderer from 'react-test-renderer';
+import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
+
+describe('Radio component tests', () => {
+  it('Radio default', () => {
+    const tree = renderer.create(<Radio value="key1" label="Default Radio" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Radio disabled', () => {
+    const tree = renderer.create(<Radio disabled value="key1" label="Disabled Radio" />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('Radio simple rendering does not invalidate styling', () => {
+    checkRenderConsistency(() => <Radio value="key1" label="Default Radio" />, 2);
+  });
+
+  it('Radio re-renders correctly', () => {
+    checkReRender(() => <Radio value="key1" label="Render twice" />);
```

Comment:
Could you add a comment that depth 2 is blocked and that we'll come back to it later?

Reply:
Yep, added!